example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 46;
+	objectVersion = 50;
```

Comment:
should we commit changes to this file?

Reply:
this is auto-generated when I run the project